app/src/globalConfig.ts

```diff
@@ -36,7 +37,9 @@ function setAppDefaultLanguage(languageId: Uuid) {
 watch(
     appLanguageIdsAsRef,
     (newVal) => {
-        localStorage.setItem("languages", JSON.stringify(newVal));
+        if (appLanguageIdsAsRef.value.includes(typeof null))
+            appLanguageIdsAsRef.value = appLanguageIdsAsRef.value.filter((id) => id !== null);
+        localStorage.setItem("languages", JSON.stringify(newVal.filter((id) => id != null)));
```

Comment:
This ensures that there are no null values in local storage if all else fails

Reply:
```suggestion
        localStorage.setItem("languages", JSON.stringify(newVal.filter((id) => id)));
```
should also work (and will work for undefined values as well)